### DISCUSSION TITLE
with_terminal() missing

### DISCUSSION THREAD

**Discussion by reula** [fonsp/Pluto.jl#935] [Questions about using Pluto] [answered]
Hi, sorry to insist, this is what I get, and I don't realize what I am doing wrong.

`with_terminal(println,"Hola")`

**UndefVarError: with_terminal not defined**

top-level scope@Local: 1

It does not find with_terminal, and I don't see it any longer in the varinfo list. 

`varinfo(PlutoUI)`

**name	size	summary**
Button	184 bytes	DataType
CheckBox	184 bytes	DataType
Clock	192 bytes	DataType
DownloadButton	192 bytes	DataType
FilePicker	184 bytes	DataType
LocalResource	0 bytes	typeof(PlutoUI.LocalResource)
MultiSelect	208 bytes	DataType
NumberField	232 bytes	DataType
PasswordField	184 bytes	DataType
PlutoUI	165.324 KiB	Module
Radio	208 bytes	DataType
RemoteResource	296 bytes	DataType
Resource	296 bytes	DataType
Select	208 bytes	DataType
Slider	240 bytes	DataType
TextField	208 bytes	DataType

**Comment by fonsp** [comment, marked as answer]
Can you share a notebook? What version of PlutoUI do you have?

**Reply by reula** [reply]
(@v1.5) pkg> status PlutoUI
Status `~/.julia/environments/v1.5/Project.toml`
  [7f904dfe] PlutoUI v0.6.2

Pluto just updated. This window does not allow me to upload the file, but it is just the following lines:

```
using PlutoUI
with_terminal() do
	println("Hola")
end
varinfo(PlutoUI)
```

BTW, is there a way to stop reactivity? 
I want to use the setting for teaching students with no knowledge of coding, and at first, sequentiality makes life simpler.

**Reply by fonsp** [reply]
Aha, you just need to update PlutoUI.

You can't disable reactivity, if sequential evaluation and terminal output is what you are looking for, then Pluto is not the right tool.
